packages/destination-actions/src/destinations/aws-s3/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import uploadCsv from './uploadCsv'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'S3 CSV Audiences',
+  slug: 'actions-s3-csv',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
```

Comment:
Remove the settings bellow because this isn't an audience destination.
```suggestion
    fields: {}
```